src/features/services/queries.ts

```diff
@@ -89,3 +90,22 @@ export function useCreateToken() {
       }),
   });
 }
+
+export function usePutValue() {
+  // eslint-disable-next-line
+  const [_, network] = useNetworkContext();
+  const queryClient = useQueryClient();
+  const { url, id } = network!;
+  const kv = new KeyValueService(url, id);
+
+  return useMutation<unknown, Error, PutValueInputs>({
+    mutationFn: async (inputs: PutValueInputs) => {
+      await kv.put(inputs);
+    },
+    onSuccess: () =>
+      // @TODO: Invalidate the query from liftedinit/roadmap#17
+      queryClient.invalidateQueries({
+        queryKey: ["", network?.url],
```

Comment:
When we save, we want to invalidate the query from the "read data" story so it's automatically fetched again.